Makefile

```diff
@@ -65,7 +65,7 @@ container-image:
 		--build-arg HUGO_VERSION=$(HUGO_VERSION)
 
 container-build: module-check
-	$(CONTAINER_RUN) $(CONTAINER_IMAGE) hugo --minify
+	$(CONTAINER_RUN) $(CONTAINER_IMAGE) sh -c "npm ci && hugo --minify"
```

Comment:
the packages should be installed into the container, I think. we should fix that ref: https://github.com/kubernetes/website/issues/22515

cc @jimangel

Reply:
@BenTheElder Yeah, I agree except the current Dockerfile doesn't COPY anything from this repo. That's actually why I put npm ci here. after mounting the volume you can execute npm command.

Reply:
What about something like an init script to run npm when the container is launched? https://success.docker.com/article/use-a-script-to-initialize-stateful-container-data

I like the idea of keeping ALL dependencies in the container if possible.

Reply:
I would not personally like to use `entrypoint.sh` with this tiny problem, though it's just my personal preference 🤔

Reply:
I think this is okay for now, and we can always iterate if we want to change it!

Reply:
👋  @inductor .
Just checking if you have the time to rebase your changes? Thank you!

Reply:
@kbhawkey I rebased it :)

Reply:
:eyes:

Reply:
I am testing the changes now.